update solar label prelim work

solderq35 opened the pull request:
![image](https://github.com/OSU-Sustainability-Office/energy-dashboard/assets/82061589/d7df3f4a-228a-41b1-a94e-28d12b75eb70)

TODO
- remove other measurements other than energy_change / energy produced from solar class (make new class?)
- maybe fix "total solar panel" etc in block.module.js's loadDefault

s-egge commented:
Making a new class with only `energy_change` didn't work, I believe due to meters not being classified as a `Solar Panel` unless they had a `total_energy` type: https://github.com/OSU-Sustainability-Office/energy-dashboard/blob/86c7cac1bf2896cc7314e6cf55505e27d9525204/backend/dependencies/nodejs/models/meter.js#L118
Removing the excess meters and changing the code to check for `energy_change` for Solar instead seems to have fixed it. I tested the Tesla solar buildings as well and they don't seem negatively affected. This has removed the extra, non-working measurements from the edit card.

solderq35 commented:
I removed a few more references to the removed measurement types:

```js
      total_energy: 'kWh',
      voltage: 'V',
      current: 'A'
 ```
 
 And updated comment in the meter_classes file to reflect the current solar meters.
 
 ##  NOTE
 I didn't remove the above measurement variables from the backend tests (`backend\tests\data_layer.spec.js` and `backend\tests\assertedData\mock_solar_data.json`, but the tests should be turned off at the moment for solar data anyways, and the solar backend testing needs a revamp (e.g. don't test with 15 minute intervals). 
 
 Low priority, push to future PR I think
